tensorflow/python/estimator/inputs/queues/feeding_functions.py

```diff
@@ -169,10 +169,17 @@ def __call__(self):
         current_epoch=self._epoch,
         total_epochs=self._num_epochs)
 
+    # Get sorted indices and remember how to reorder output.
```

Comment:
The code looks good. But can you add a comment to briefly explain why this is required?